### DISCUSSION TITLE
CREW SUPPLY Oxygen

### DISCUSSION THREAD

**Discussion by MStallen** [flybywiresim/aircraft#8716] [Help] [answered]
Is it correct that the only light after startup which is on at the overheadpanel is the crewsupply oxygen light?

**Comment by yourcptainspeaking** [comment, marked as answer]
The off light on the crew oxygen supply PB should be extinguished as the system is supposed to be turned on prior to startup. A good rule of thumb for normal airbus operations is that there should be no white lights visible on the overhead unless a checklist item or ECAM action leads to one turning on.

**Reply by MStallen** [reply]
Thank you, it was not in my checklist so I was not sure.
